pkg/apis/smith/v1/types.go

```diff
@@ -150,7 +172,8 @@ type BundleCondition struct {
 // +k8s:deepcopy-gen=true
 type BundleStatus struct {
 	// Represents the latest available observations of a Bundle's current state.
-	Conditions []BundleCondition `json:"conditions,omitempty"`
+	Conditions     []BundleCondition `json:"conditions,omitempty"`
+	ResourceStatus []ResourceStatus  `json:"resourceStatuses,omitempty"`
```

Comment:
Hmm... I guess this is some style thing that this is ResourceStatus rather than ResourceStatuses. But still, confusing.

Reply:
Yes, I was following style of some code from Kubernetes.